src/main/java/org/session13/homework/librarymanagementsystem/services/MemberServices.java

```diff
@@ -0,0 +1,44 @@
+package org.session13.homework.librarymanagementsystem.services;
+
+import org.session13.homework.librarymanagementsystem.librarymanagement.Book;
+import org.session13.homework.librarymanagementsystem.librarymanagement.Library;
+import org.session13.homework.librarymanagementsystem.profiles.Member;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+public class MemberServices {
```

Comment:
Currently, MemberServices handles both the borrowing and returning of books. Consider separating these responsibilities into different methods or classes to adhere to the Single Responsibility Principle.